docs/webpack-helpers.md

```diff
@@ -1,154 +0,0 @@
-### Table of Contents
```

Comment:
(Repeating the comment from above as it seems to have been read over)

I think it would be better to move this content into the wiki to keep everything together a bit more. Currently you have to switch between this doc and the wiki for some config information which is a bit awkward in my experience.

Certainly open to other thoughts, maybe we should move the common config recipes into a doc instead? Not sure.